stats/research.json

```diff
@@ -8153,7 +8153,8 @@
         "id": "R-Wpn-Missile-Damage02",
         "name": "Advanced Missile Warhead Mk2",
         "requiredResearch": [
-            "R-Wpn-Missile2A-T"
+            "R-Wpn-Missile-Damage01",
+            "R-Wpn-MdArtMissile"
```

Comment:
In line 8157 "R-Wpn-MdArtMissile" has to be replaced by "R-Defense-MdArtMissile"

Reply:
Forget it. Read my comment in issue #31